src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -695,5 +695,77 @@ void PostStepsList::add_convert_layout_impl(const std::vector<uint64_t>& dims) {
         },
         "convert layout " + vector_to_string(dims));
 }
+
+void PostStepsList::add_convert_color_impl(const ColorFormat& dst_format) {
+    m_actions.emplace_back(
+        [dst_format](const Output<Node>& node, PostprocessingContext& context) {
+            if (context.color_format() == dst_format) {
+                return std::make_tuple(node, false);
+            }
+            if ((context.color_format() == ColorFormat::RGB || context.color_format() == ColorFormat::BGR) &&
+                (dst_format == ColorFormat::RGB || dst_format == ColorFormat::BGR)) {
+                auto res = reverse_channels({node}, context);
+                context.color_format() = dst_format;
+                return res;
+            }
+            OPENVINO_ASSERT(false,
+                            "Source color format '",
+                            color_format_name(context.color_format()),
+                            "' is not convertible to '",
+                            color_format_name(dst_format),
+                            "'");
```

Comment:
```suggestion
            OPENVINO_THROW(
                            "Source color format '",
                            color_format_name(context.color_format()),
                            "' is not convertible to '",
                            color_format_name(dst_format),
                            "'");
```

Reply:
Is `OPENVINO_THROW` any benefit? 
I follow current file's behavior to use `OPENVINO_ASSERT` to keep the same code style.

Reply:
Yes, OPENVINO_THROW is simpler in case there is not need to do some check the additional `if` want be added by put dummy `false` condition (OPENVINO_THROW produces smaller bin-size code). Condition from OPENVINO_ASSERT is converted to string for error message.

Reply:
OK, will change to `OPENVINO_THROW`, but not to modify old code by replacing `OPENVINO_ASSERT with `OPENVINO_THROW`